scripts/cc-cli/src/utils/account.ts

```diff
@@ -13,19 +13,37 @@ export function initECDSAKeyringPairFromPK(pk: string) {
   return pair;
 }
 
-export async function getStashSeedFromEnvOrPrompt() {
+export async function getStashSeedFromEnvOrPrompt(interactive: boolean) {
+  if (!interactive && !process.env.CC_STASH_SEED) {
+    console.error(
+      "Error: Must specify a seed phrase for the Stash account in the environment variable CC_STASH_SEED or use an interactive shell."
+    );
+    process.exit(1);
+  }
```

Comment:
IMO refactor error handling in `getSeedFromEnvOrPrompt()` and `throw` based on the return value instead of pushing the env var check here. 

Same for all of the functions below.

Reply:
From what I can see `getSeedFromEnvOrPrompt()` is not throwing an exception but still calling `process.exit` if it fails. 

Was it supposed to be refactored or not?  B/c it the error handling and exiting is part of `getSeedFromEnvOrPrompt()` then none of it's callers need to do any error handling really.

Reply:
Currently `getSeedFromEnvOrPrompt` is throwing errors and the specific ones such as `getCallerSeed...` are doing the handling. Would you rather have the handling up in the actual commands?

Recent refactor also moved the env check and interactive flag check to `getSeedFromEnvOrPrompt`.

Reply:
Grrr. The proper code didn't show up for me so I was looking at an old version.